src/View/Requirements_Backend.php

```diff
@@ -238,11 +238,12 @@ public function setAssetHandler(GeneratedAssetHandler $handler)
     /**
      * Gets the minification service for this backend
      *
-     * @deprecated 4.0.0:5.0.0
+     * @deprecated 4.0.0
      * @return Requirements_Minifier
      */
     public function getMinifier()
     {
+        Deprecation::notice('4.12.0');
```

Comment:
```suggestion
        Deprecation::notice('4.0.0');
```
Also add a message here and to the phpdoc

Reply:
Updated